js/plugins/tabs.html

```diff
@@ -90,7 +90,7 @@
             var type = $('input[name=type]:checked').val();
             var mparent = $('#sl_parent').val();
             var content = $('#txt_content').val();
-            var id = 'bs_tab' + e.timeStamp;
+            var id = 'bs_tab' + guid();
```

Comment:
Suggestion to add an underscore after 'bs_tab' for the id. So, `var id = 'bs_tab_' + guid();`